Objects/listobject.c

```diff
@@ -2737,8 +2748,11 @@ list_index_impl(PyListObject *self, PyObject *value, Py_ssize_t start,
             stop = 0;
     }
     for (i = start; i < stop && i < Py_SIZE(self); i++) {
```

Comment:
The `i < Py_SIZE(self)` is probably redundant now.

Reply:
Yeah! Nice improvement :)